git_day_1.adoc

```diff
@@ -0,0 +1,138 @@
+Git Gud Speaker Outline - Day 1
```

Comment:
Asciidoc (perhaps it is specific to AsciiDoctor) has a standard for defining the document title / primary header.

```suggestion
= Git Gud Speaker Outline - Day 1
```

Reply:
Fixed